insights/parsers/ros_config.py

```diff
@@ -78,63 +79,34 @@
 parse = LogSpecs << EOF
```

Comment:
This can be removed since parse is redefined later

Reply:
Sure. Also @csams in your branch, you haven't defined a subclass [here](https://github.com/RedHatInsights/insights-core/blob/ros/insights/parsers/pmlogger.py). So, where should I place my class, is it correct [where](https://github.com/RedHatInsights/insights-core/pull/3197/files#diff-2c2c758bef3caa206db8e25a1774b322e1396aaaac276dd4d7f802f4177539a9R110) it is now: ?

Reply:
@apuntamb I didn't give much thought to names/organization or connecting the parsing logic to the rest of insights. Since the grammar is specific to `pmlogger`, it makes sense to me to name the module `pmlogger` and then put specific `@parser` classes that use it into the same file, e.g. `ROSConfig(Parser)`, etc.

Reply:
@csams I'm not sure if I understand this but what I meant was currently running `insights inspect insights.parsers.ros_config.RosConfig` gives me this error `object of type 'NoneType' has no len()`. So, is it because of how I have defined the class or is it related to something else?

Reply:
@apuntamb you need to use the `@parser(Specs.ros_config)` decorator on your parser class below.

Reply:
+1
Thanks @bfahr ! That worked. The tests are green too. Please let me know if there is a better or sophisticated way of writing the tests as well.